toolchain/sem_ir/dump.cpp

```diff
@@ -247,7 +247,21 @@ LLVM_DUMP_METHOD auto Dump(const File& file,
 LLVM_DUMP_METHOD auto Dump(const File& file, SpecificId specific_id) -> void {
   DumpNoNewline(file, specific_id);
   llvm::errs() << '\n';
-  Dump(file, file.specifics().Get(specific_id).args_id);
+  if (specific_id.has_value()) {
+    Dump(file, file.specifics().Get(specific_id).args_id);
+  }
+}
+
+LLVM_DUMP_METHOD auto Dump(const File& file,
+                           SpecificInterfaceId specific_interface_id) -> void {
+  const auto& interface = file.specific_interfaces().Get(specific_interface_id);
+  llvm::errs() << specific_interface_id;
+  llvm::errs() << '\n';
+  llvm::errs() << "  - interface: ";
```

Comment:
I'll suggest leaning into the streaming API, i.e.:

```suggestion
  llvm::errs() << specific_interface_id << "\n  - interface: ";
```

Reply:
Done - partway. I combined the `\n` but kept each line as a separate statement in the code, as I find that a lot easier to map to what it will look like as the output, and to verify you have the same/correct number of spaces before the `-` and such.